src/web/app/components/adding-question-panel/adding-question-panel.component.html

```diff
@@ -20,11 +20,11 @@
       </ng-container>
     </div>
   </div>
-  <a type="button" class="btn btn-link" *ngIf="!isLinkDisabled" target="_blank" rel="noopener noreferrer"
+  <a type="button" class="btn btn-link p-1 p-sm-3" *ngIf="!isLinkDisabled" target="_blank" rel="noopener noreferrer"
     tmRouterLink="/web/instructor/help" [queryParams]="{questionId: 'questions', section: 'questions'}" aria-label="Help for questions">
     <i class="fas fa-info-circle"></i>
   </a>
-  <a type="button" class="btn btn-link" *ngIf="isLinkDisabled"><i class="fas fa-info-circle"></i></a>
-  <button id="btn-copy-question" class="btn btn-primary" (click)="copyQuestionsFromOtherSessionsHandler()"
+  <a type="button" class="btn btn-link p-1 p-sm-3" *ngIf="isLinkDisabled"><i class="fas fa-info-circle"></i></a>
```

Comment:
Let's have the padding apply to just left and right, so the changes should be `px-1 px-sm-3` for both